### ISSUE TITLE
 language in UI to German not set

### ISSUE THREAD

**Issue by ace16vitamine** [jokob-sk/Pi.Alert#419]
Built on: 2023-09-10 | Version: v23.9.10

I changed the language in UI to German but the language is still EN.

In the config file option is set: UI_LANG='German'

**Comment by cvc90** [comment]
Hi @ace16vitamine,

I have detected an error in the .json file in the "German" version

A [PR 429](#429) has already been sent with the solution to this problem.

**Comment by ace16vitamine** [comment]
Top, thanks !!!

**Comment by jokob-sk** [comment]
Thanks @cvc90 🙏

@ace16vitamine can you check the dev image if this issue is fixed?

```
version: "3"
services:
  pialert:
    container_name: pialert
    image: "jokobsk/pi.alert_dev"
    network_mode: "host"
    restart: always
    volumes:
      - ${APP_DATA_LOCATION}/pialert_dev/config:/home/pi/pialert/config
      - ${APP_DATA_LOCATION}/pialert_dev/db:/home/pi/pialert/db
      - ${LOGS_LOCATION}/:/home/pi/pialert/log
    environment:
      - TZ=Europe/Berlin
      - HOST_USER_ID=1036
      - HOST_USER_GID=1000
      - PORT=20211
```

**Comment by ace16vitamine** [comment]
Confirmed, it is German.  :-)

**Comment by jokob-sk** [comment]
Thanks 🙏 :)

**Comment by jokob-sk** [comment]
Keeping it open for others to find

**Comment by jokob-sk** [comment]
Released in today's hotfix (23.9.11)
